README.md

```diff
@@ -3,7 +3,7 @@
 <div align="center">
 
   <h2><b>BudgetBuddy</b></h2>
-  <h4>Luis Emilio Rojas</h4>
+  <h4>Luis Emilio Rojas Rojas</h4>
 
 </div>
```

Comment:
While you did a good job working on the project requirements Im afraid you did not follow design guidleines I know you want to perfect your work but it is also important to follow design rules, so it would be nice to follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

This is your splash screen:

![Screenshot 2023-11-26 at 5 29 04 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/02b0148f-0b1c-4aa4-b3fd-dab2b31dd8a7)

Your splash screen should look like this:

![Screenshot 2023-11-26 at 5 10 20 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/fb45c693-5389-4a1a-95be-00694ccdb37c)

Here are both your categories page and transaction pages:

![Screenshot 2023-11-26 at 5 20 57 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/cbe62cd6-711e-427a-956c-a4600119b7f8)
![Screenshot 2023-11-26 at 5 21 21 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/de6590be-72a6-4606-8a28-bc430eb67184)
![Screenshot 2023-11-26 at 5 23 06 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/fc358bfb-f461-4d9f-8a03-a726314d1caa)
![Screenshot 2023-11-26 at 5 23 15 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/b21eb965-00c2-47d6-911d-49f3dbca1974)
![Screenshot 2023-11-26 at 5 21 47 AM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/48747b6c-a96a-41df-9ed6-3968fc0f29d4)

Please follow this design:

![Screenshot 2023-11-25 at 7 34 39 PM](https://github.com/lerfast/Module5-Capstone-budget-app/assets/79506105/2fd5436d-3a3f-4fd3-970e-032559364590)

its says in the project requirements that:

Transactions page


For a given category, the list of transactions is presented ordered by the most recent
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category)
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction
When the user clicks on the "Back" button (<), the user navigates to the home page


"Add a new category" page


The user fills out a form to create a new category, indicating their name and icon (both mandatory)
The user click (or taps) the "Save" button to create the new category, and it's taken to the home page on success
When the user clicks on the "Back" button (<), the user navigates to the home page


"Add a new transaction" page


The user fills out a form to create a new transaction with:

name (mandatory)
amount (mandatory)
categories (mandatory at least one)

It is very important to pay attention to design requirements. Please pay attention to detail.